core/modules/editor/operations/text/prefix-lines.js

```diff
@@ -25,8 +25,8 @@ exports["prefix-lines"] = function(event,operation) {
 	$tw.utils.each(lines,function(line,index) {
 		// Remove and count any existing prefix characters
 		var count = 0;
-		while(line.charAt(0) === event.paramObject.character) {
-			line = line.substring(1);
+		while(line.startsWith(event.paramObject.character)) {
```

Comment:
Hi @MaxGyver83 I am afraid it's a bit of a nuisance, but the core currently sticks to ECMAScript5.1, which doesn't include `.startsWith`.

Reply:
> Oh, sorry, I didn't know. I have created a `startsWith` function in `core/modules/utils/utils.js`.

Awesome, thanks.

Reply:
My pleasure!